tests/unit/geo.tcl

```diff
@@ -535,4 +539,42 @@ start_server {tags {"geo"}} {
         set test_result
     } {OK}
     }
+
+    test {GEOSEARCH fuzzy test} {
```

Comment:
that's a good test, but i think we need to extend it.
currently you only have `lat_delta`, so you only test the top edge of the trapezoid.
i think you should at the very least test also:
1. a horizontal edge (i.e. have a `long_delta`
2. try to apply both at the same time, so that you test the corners.

ideally, we should test:
1. a point inside and a point outside at the center of each of the edges. (total of 8 tests)
2. 4 points around each of the corners (total of 16 tests).

but it may be sufficient to just test one corner, and two edges (horizontal one and vertical one).

Reply:
update: 
The latest code generates 8 points, which are located at each vertex and the center of each side.